### PR TITLE
docs(docs/features/software-templates/writing-custom-field-extensionsmd): add pending imports.

### PR DESCRIPTION
**Pull request by padupe**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/features/software-templates/writing-custom-field-extensions.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ import React from 'react';
 import { FieldExtensionComponentProps } from '@backstage/plugin-scaffolder-react';
 import type { FieldValidation } from '@rjsf/utils';
 import FormControl from '@material-ui/core/FormControl';
+import {
+  FormControl,
```
Comment (Parsifal-M) [inline docs/features/software-templates/writing-custom-field-extensions.md:37]: I think we don't need this one right? Shouldn't it be:

```tsx
import { InputLabel, Input, FormHelperText } from '@material-ui/core';
```

File: docs/features/software-templates/writing-custom-field-extensions.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ import React from 'react';
 import { FieldExtensionComponentProps } from '@backstage/plugin-scaffolder-react';
 import type { FieldValidation } from '@rjsf/utils';
 import FormControl from '@material-ui/core/FormControl';
```
[the remark this answers is not in the document]
Comment (padupe) [reply docs/features/software-templates/writing-custom-field-extensions.md]: Really... my fault lol
Thanks for the tip!

**Review by jhaals — CHANGES_REQUESTED** [review]
Thanks for the fix! Looks like there's a bunch of unrelated yarn related CI errors. Could you please try to rebase your branch on top of an updated master branch

**Comment by padupe** [comment]
@jhaals I apologize for the delay, but I have already updated my branch as instructed.

**Review by jhaals — APPROVED** [review]

File: docs/features/software-templates/writing-custom-field-extensions.md [hunk 3]
```diff
@@ -32,7 +32,12 @@ As an example, we will create a component that validates whether a string is in
 import React from 'react';
 import { FieldExtensionComponentProps } from '@backstage/plugin-scaffolder-react';
 import type { FieldValidation } from '@rjsf/utils';
-import FormControl from '@material-ui/core/FormControl';
+import {
+  FormControl,
+  FormHelperText,
+  Input,
+  InputLabel,
+} from '@material-ui/core';
```
Comment (jhaals) [inline docs/features/software-templates/writing-custom-field-extensions.md:40]: Sorry I realized that top level imports are not allowed. 
So import needs to be in the form of `import FormControl from '@material-ui/core/FormControl';` `import InputLabel from '@material-ui/core/InputLabel';` and so on.

**Review by jhaals — CHANGES_REQUESTED** [review]

File: docs/features/software-templates/writing-custom-field-extensions.md (hunk 3 above)
Comment (padupe) [reply docs/features/software-templates/writing-custom-field-extensions.md]: Don't worry! Adjustment done!

**Review by jhaals — APPROVED** [review]
Thanks!

**Pull request merged**